### PR TITLE
Improves signaler and bomb logging

### LINKED ISSUE #12592: Bomb logging needs improvement
Soca knows more about this since it was when she was admining but

Bomb logging seems to be a mess where a decent deal of information is either missing or not properly explained.

Examples would be:
[15:53:24]GAME: Bomb valve opened in <A HREF='?src=holder;adminplayerobservecoodjump=1;X=253;Y=197;Z=1'>Courtroom</a> with the remote signaling device attacher: Mitchell Owen(pickledheretic)(<A HREF='?src=holder;adminmoreinfo=[0x30002cf]'>?</A>) Last touched by: RedCotober(<A HREF='?src=holder;adminmoreinfo=[0x30002ef]'>?</A>

[15:53:24]ADMINWARN: <span class="admin"><span class="prefix">ADMIN LOG:</span> <span class="message">Bomb valve opened in <A HREF='?src=holder;adminplayerobservecoodjump=1;X=253;Y=197;Z=1'>Courtroom</a> with the remote signaling device attacher: Mitchell Owen(pickledheretic)(<A HREF='?src=holder;adminmoreinfo=[0x30002cf]'>?</A>) Last touched by: RedCotober(<A HREF='?src=holder;adminmoreinfo=[0x30002ef]'>?</A>)</span></span>

Where PickledHeretic made and sold the bombs, however Red October activated them? However it pings it towards PickledHeretic, perhaps because he's the one who installed it?

I'm honestly unsure but I feel like it could be improved a bit.

### LINKED ISSUE #14350: fix bomb logging
It sucks.

ADMIN LOG: Bomb valve opened in Thunderdome with the remote signaling device attacher: Unknown Last touched by: zealotron(?)

Syndie-spawned bomb displays this.

It was triggered by Arch Sunder.

It's like that for every bomb, it only tells us who put the signaler on and who touched it last, not who actually made it blow up

### PR DESCRIPTION
**Pull request by EdXbtkOEwGw**
Signaler logging is now done in the Investigate->Wires tab, which also permanently puts them in the public logs.
Warning message when a bomb explodes now has a lot more information that gives you a better idea what caused it.
 
Fixes #14350
Fixes #12592

### CODE REVIEW

**Comment by SonixApache** [comment]
#14350
#12592

**Comment by EdXbtkOEwGw** [comment]
Thanks

**Comment by SGuivenne** [comment]
As someone who has been banned for griefing with bombing escape.  I did note that there was a LOT of confusion in regard to voice detonators on bombs.  Will this fix that or should these things also log detonation along with trigger word listed in said detonation log so that its easier to figure out who actually set off the bombs.  

It really shouldn't take 10 minutes to figure out which asshole blew up the bomb.

**Comment by EdXbtkOEwGw** [comment]
It will explicitly tell you that a voice detonator did it, who attached the voice detonator (which should be the person to get bent anyways), and the last user processed which will probably be who said the trigger word but may not necessarily be if telecomms is delayed or some convoluted shit.
I think this is pretty good.

File: code/game/objects/items/devices/transfer_valve.dm [hunk 1]
```diff
@@ -178,30 +176,24 @@
...
-			attacher_name = "Unknown"
-		else
-			attacher_name = "[attacher.name]([attacher.ckey])"
-
-		var/log_str = "Bomb valve opened in <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[bombturf.x];Y=[bombturf.y];Z=[bombturf.z]'>[A.name]</a> "
-		log_str += "with [attached_device ? attached_device : "no device"] attacher: [attacher_name]"
-
-		if(attacher)
-			log_str += "(<A HREF='?_src_=holder;adminmoreinfo=\ref[attacher]'>?</A>)"
+		var/log_str = "Tank transfer valve opened in <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[bombturf.x];Y=[bombturf.y];Z=[bombturf.z]'>[A.name]</a>, "
```
Comment (DamianX) [inline code/game/objects/items/devices/transfer_valve.dm]: Could use `formatJumpTo` instead

**Comment by Fruktlimpa** [comment]
God bless, bomb logging was horrendous.

**Pull request merged**